start.sh

```diff
@@ -31,9 +31,18 @@ trap cleanup EXIT SIGINT SIGTERM
 
 # Main execution
 main() {
+    # make vcan0 interface
```

Comment:
I would actually remove this from here

Reply:
We have the instructions to set this script up

Reply:
removed this, as discussed.